github/ci/prow-deploy/files/jobs/kubevirt/containerdisks/containerdisks-presubmits.yaml

```diff
@@ -87,6 +87,43 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    optional: false
+    annotations:
+      testgrid-create-test-group: "false"
+    cluster: prow-s390x-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-podman-in-container-enabled: "true"
+      preset-docker-mirror-proxy: "true"
```

Comment:
I think you will need to remove this preset - the docker-mirror-proxy is not available in the s390x env

Reply:
Thanks @brianmcarey . Preset removed.